kong/plugins/aws-lambda/handler.lua

```diff
@@ -83,7 +83,11 @@ function AWSLambdaHandler:access(conf)
     return responses.send_HTTP_INTERNAL_SERVER_ERROR(err)
   end
 
-  ngx.status = res.status
+  if headers['X-Amzn-Function-Error'] == 'Unhandled' and conf.unhandled_response ~= -1 then
```

Comment:
Can someone confirm the upgrade path this plugin would take? I'm imagining it'd get the default value `-1` but I couldn't confirm before submitting this PR.